workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -546,8 +639,8 @@ PlayerSettings:
   webGLLinkerTarget: 0
   scriptingDefineSymbols:
     1: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP
-    4: UNITY_POST_PROCESSING_STACK_V2
-    7: UNITY_POST_PROCESSING_STACK_V2
+    4: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP
+    7: UNITY_POST_PROCESSING_STACK_V2;UNITY_DISABLE_AUTOMATIC_SYSTEM_BOOTSTRAP
```

Comment:
Why is this added back in

Reply:
I think this was accidental, I'll revert this

Reply:
@samiwh what do you mean back in? It's still present in master for PC. This just adds the symbol for Android and iOS.

Reply:
why did this got removed?

Reply:
agreeing with @DanailPenev  here, we need these symbols for mobile support.

Reply:
Out of interest what breaks then if you don't enable the automatic bootstrap on mobile?

Reply:
none of the systems in the playground are annotated with `DisableAutoCreation`. You might remember the discussion we had back then when deciding whether to have a default world or not. Those will be added to the default world. There are a few systems in the playground that also use the WorkerSystem. Those will throw errors without that symbol

Reply:
in general: Either we have the automatic bootstrap enabled for all platforms or for none. Having it only for a few just seems wrong and could introduce issues without us realizing it